### PR TITLE
Pinned OpenVINO as a shared library

### PR DESCRIPTION
**Pull request by ilya-lavrenov**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
```
ValueError: Variant configuration errors in /home/conda/recipe_root/conda_build_config.yaml:
  libopenvino-dev key contains an invalid character '-'
```
Why `-` is invalid sign?

**Comment by hmaarrfk** [comment]
Replace it with underscore

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -507,6 +507,8 @@ libopentelemetry_cpp:
   - '1.13'
 libosqp:
   - 0.6.3
+libopenvino-dev:
```
Comment (hmaarrfk) [inline recipe/conda_build_config.yaml:510]:
```suggestion
libopenvino_dev:
```

this is a quirk of this pinning since(i think)  "-" is special in yaml

**Comment by hmaarrfk** [comment]
thanks!

**Comment by ilya-lavrenov** [comment]
BTW, `libopenvino-dev` is a meta package, which provides `libopenvino` via `run_exports`:
- `libopenvino-dev` is used in recipes
- `libopenvino` is added to final packages as a dependency (e.g. to `libopencv`)

What package should be pinned here? `libopenvino` or `libopenvino-dev` ?
Originally I supposed that `libopenvino-dev`, but now I'm starting to think of `libopenvino`

**Comment by xhochy** [comment]
`-dev` should be listed here as this is the one used in recipes.

**Comment by hmaarrfk** [comment]
> `-dev` should be listed here as this is the one used in recipes.

Indeed. This is something I checked when I added the automerge tag

**Pull request merged**